### DISCUSSION TITLE
About Fl_Group::group_resize_depth_

### DISCUSSION THREAD

**Discussion by Albrecht-S** [fltk/fltk#999] [Development]
**_Summary:_** Issue #987 deals with subwindows under Wayland.
- The proposed patch in https://github.com/fltk/fltk/issues/987#issuecomment-2198482832 suggests to increment and decrement a recently introduced static variable in `Fl_Group` before and after resizing children of `Fl_Flex`.
- In https://github.com/fltk/fltk/issues/987#issuecomment-2198600468 I wrote that I consider this a bad approach.
- In https://github.com/fltk/fltk/issues/987#issuecomment-2199364632 @ManoloFLTK explained why this approach was used in `Fl_Group` and the window and Wayland drivers and why he suggests the patch mentioned above.
- FYI: This new variable was introduced in Git commit 1c6a0c1a8fb254945179276ecbd359cc1926fbb1.

I'm moving the discussion of this aspect of issue #987 to this discussion to avoid going off-topic in this issue and to concentrate on the new variable and to try finding a better approach.

To avoid duplicating all arguments please follow the links above.

**Comment by ManoloFLTK** [comment]
@Albrecht-S I've now found how to go without change to class Fl_Group at all. Changes to this class (and Fl_Flex) have been removed at commit b551644. I suggest to close this conversation.

**Reply by Albrecht-S** [reply]
Thanks, good job. Closing this discussion.

**Comment by erco77** [comment]
Oh, is this using the new github discussions?

I was wondering why I was getting weird email forwards from the fltk.org backend scripts, indicating a new message type needs to be integrated.

Need to figure out where these messages should go. Probably a new fltk.discussions newsgroup.

**Reply by Albrecht-S** [reply]
@erco77 Yes, sorry that I missed to inform you. Finally I made an announcement in fltk.general.

For now I wouldn't investigate much time in mirroring GitHub Discussions in a newsgroup though.
